core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.html

```diff
@@ -1,12 +1,22 @@
 <div class="audio-header" ng-if="isAudioBarAvailable()">
   <div ng-if="audioBarIsExpanded" class="audio-controls">
-    <div style="width: 560px;">
-      <i class="fas audio-controls-button-icon"
-         ng-click="onPlayButtonClicked()"
-         ng-class="{'fa-ellipsis-h': audioLoadingIndicatorIsShown, 'fa-play-circle': !isAudioPlaying(), 'fa-pause-circle': isAudioPlaying(), 'audio-controls-audio-not-available': !isAudioAvailableInCurrentLanguage() || audioIsLoading}"
-         uib-tooltip="<[!isAudioAvailableInCurrentLanguage() ? ('I18N_PLAYER_AUDIO_NOT_AVAILABLE_IN' | translate:{languageDescription:getCurrentAudioLanguageDescription()}) : '']>"
-         tooltip-append-to-body="true" tooltip-placement="right">
-      </i>
+    <div class="audio-header-control-buttons">
+      <div class="audio-header-control-icons">
+        <span class="oppia-replay-white-circle" ng-click="onBackwardButtonClicked()" ng-if="progressBarIsShown">
+          <i class="fas fa-undo audio-undo-icon"></i>
+          <span class="oppia-five-icon">5</span>
+        </span>
+        <i class="fas audio-controls-button-icon"
+           ng-click="onPlayButtonClicked()"
```

Comment:
Do not handle click on `i` tag either use button or `anchor <a>` tag, you can do like this:
`<a> <i>...</i></a>`

This is important from a11y perspective as well as you won't see any changes in cursor when you'll navigate to this button (generally we expect pointer cursor form buttons/clickable elements)

Reply:
Done